.gitignore

```diff
@@ -13,11 +13,14 @@ coverage
 .nyc_output
 junit.xml
 
+# IDE-related files and directories
```

Comment:
Oh, I didn't know about the global option, that makes sense. I'll probably rebase away this commit since I only added it in the first place to be able to run the codemod, which wouldn't start with uncommitted files present.